### PR TITLE
Bump v1.0.0rc4

### PR DESCRIPTION
**Pull request by xiexinch**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMDet3D.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,11 +62,17 @@ The 1.x branch works with **PyTorch 1.6+**.
 
 ## What's New
 
-v1.0.0rc3 was released in 31/12/2022.
+v1.0.0rc4 was released in 01/30/2023.
```
Comment (MeowZheng) [inline README.md:65]:
```suggestion
v1.0.0rc4 was released on 30/01/2023.
```

File: README.md [hunk 2]
```diff
@@ -62,11 +62,17 @@ The 1.x branch works with **PyTorch 1.6+**.
...
-- Support test time augmentation ([#2184](https://github.com/open-mmlab/mmsegmentation/pull/2184))
-- Add 'Projects/' folder and the first example project ([#2412](https://github.com/open-mmlab/mmsegmentation/pull/2412))
+- Add nnUNet-style Gaussian Noise and Blur ([#2373](https://github.com/open-mmlab/mmsegmentation/pull/2373))
+- Add BioMedicalRandomGamma ([#2406](https://github.com/open-mmlab/mmsegmentation/pull/2406))
+- Add BioMedical3DPad ([#2383](https://github.com/open-mmlab/mmsegmentation/pull/2383))
+- Add ISNet in projects ([#2400](https://github.com/open-mmlab/mmsegmentation/pull/2400))
+- Support synapse dataset ([#2432](https://github.com/open-mmlab/mmsegmentation/pull/2432), [#2465](https://github.com/open-mmlab/mmsegmentation/pull/2465))
+- Support HieraSeg on cityscapes ([#2444](https://github.com/open-mmlab/mmsegmentation/pull/2444))
+- Support Mapillary Vistas Dataset ([#2484](https://github.com/open-mmlab/mmsegmentation/pull/2484))
+- Add BioMedical3DRandomFlip ([#2404](https://github.com/open-mmlab/mmsegmentation/pull/2404))
```
Comment (MeowZheng) [inline README.md:75]:
```suggestion
- Add Gaussian Noise and Blur for biomedical data ([#2373](https://github.com/open-mmlab/mmsegmentation/pull/2373))
- Add BioMedicalRandomGamma ([#2406](https://github.com/open-mmlab/mmsegmentation/pull/2406))
- Add BioMedical3DPad ([#2383](https://github.com/open-mmlab/mmsegmentation/pull/2383))
- Add BioMedical3DRandomFlip ([#2404](https://github.com/open-mmlab/mmsegmentation/pull/2404))
- Add ISNet in projects ([#2400](https://github.com/open-mmlab/mmsegmentation/pull/2400))
- Support synapse dataset ([#2432](https://github.com/open-mmlab/mmsegmentation/pull/2432), [#2465](https://github.com/open-mmlab/mmsegmentation/pull/2465))
- Support HieraSeg on cityscapes ([#2444](https://github.com/open-mmlab/mmsegmentation/pull/2444))
- Support Mapillary Vistas Dataset ([#2484](https://github.com/open-mmlab/mmsegmentation/pull/2484))
```

File: docs/en/notes/changelog.md [hunk 3]
```diff
@@ -1,5 +1,40 @@
...
+
+- Add nnUNet-style Gaussian Noise and Blur ([#2373](https://github.com/open-mmlab/mmsegmentation/pull/2373))
+- Add BioMedicalRandomGamma ([#2406](https://github.com/open-mmlab/mmsegmentation/pull/2406))
+- Add BioMedical3DPad ([#2383](https://github.com/open-mmlab/mmsegmentation/pull/2383))
+- Add ISNet in projects ([#2400](https://github.com/open-mmlab/mmsegmentation/pull/2400))
+- Support synapse dataset ([#2432](https://github.com/open-mmlab/mmsegmentation/pull/2432), [#2465](https://github.com/open-mmlab/mmsegmentation/pull/2465))
+- Add `gt_edge_map` field to SegDataSample ([#2466](https://github.com/open-mmlab/mmsegmentation/pull/2466))
+- Support HieraSeg on cityscapes ([#2444](https://github.com/open-mmlab/mmsegmentation/pull/2444))
+- Support Mapillary Vistas Dataset ([#2484](https://github.com/open-mmlab/mmsegmentation/pull/2484))
+- Add BioMedical3DRandomFlip ([#2404](https://github.com/open-mmlab/mmsegmentation/pull/2404))
```
Comment (MeowZheng) [inline docs/en/notes/changelog.md:15]: add #2516  #2517  #2518

**Review by MeowZheng — COMMENTED** [review]
I think mmseg/__init__.py should assert mmcv == 2.0.0rc3

File: docs/en/notes/faq.md [hunk 4]
```diff
@@ -6,33 +6,34 @@ We list some common troubles faced by many users and their corresponding solutio
...
-|         0.11.0         |  mmcv-full>=1.1.4, \<1.3.0  |            Not required             |          Not required          |
-|         0.10.0         |  mmcv-full>=1.1.4, \<1.3.0  |            Not required             |          Not required          |
-|         0.9.0          |  mmcv-full>=1.1.4, \<1.3.0  |            Not required             |          Not required          |
-|         0.8.0          |  mmcv-full>=1.1.4, \<1.2.0  |            Not required             |          Not required          |
-|         0.7.0          |  mmcv-full>=1.1.2, \<1.2.0  |            Not required             |          Not required          |
-|         0.6.0          |  mmcv-full>=1.1.2, \<1.2.0  |            Not required             |          Not required          |
+| MMSegmentation version |          MMCV version          | MMClassification (optional) version | MMDetection (optional) version |
+| :--------------------: | :----------------------------: | :---------------------------------: | :----------------------------: |
+|        1.0.0rc4        |        mmcv == 2.0.0rc3        |           mmcls>=1.0.0rc0           | mmdet>=3.0.0rc4, \<=3.0.0rc5>  |
```
Comment (MeowZheng) [inline docs/en/notes/faq.md:11]:
```suggestion
| :--------------------: | :----------------------------: | :---------------------------------: | :----------------------------: |
|        1.x/dev-1.x branch        |        mmcv == 2.0.0rc3        |           mmcls>=1.0.0rc0           | mmdet>=3.0.0rc4, \<=3.0.0rc5>  |
|        1.0.0rc4        |        mmcv == 2.0.0rc3        |           mmcls>=1.0.0rc0           | mmdet>=3.0.0rc4, \<=3.0.0rc5>  |
```

File: docs/en/notes/changelog.md [hunk 5]
```diff
@@ -1,5 +1,46 @@
...
+
+### Features
+
+- Add Gaussian Noise and Blur for biomedical data ([#2373](https://github.com/open-mmlab/mmsegmentation/pull/2373))
+- Add BioMedicalRandomGamma ([#2406](https://github.com/open-mmlab/mmsegmentation/pull/2406))
+- Add BioMedical3DPad ([#2383](https://github.com/open-mmlab/mmsegmentation/pull/2383))
+- Add BioMedical3DRandomFlip ([#2404](https://github.com/open-mmlab/mmsegmentation/pull/2404))
+- Add `gt_edge_map` field to SegDataSample ([#2466](https://github.com/open-mmlab/mmsegmentation/pull/2466))
+- Support synapse dataset ([#2432](https://github.com/open-mmlab/mmsegmentation/pull/2432), [#2465](https://github.com/open-mmlab/mmsegmentation/pull/2465))
+- Support Mapillary Vistas Dataset ([#2484](https://github.com/open-mmlab/mmsegmentation/pull/2484))
```
Comment (MeowZheng) [inline docs/en/notes/changelog.md:18]:
```suggestion
- Support Mapillary Vistas Dataset in projects ([#2484](https://github.com/open-mmlab/mmsegmentation/pull/2484))
```

**Review by MengzhangLI — APPROVED** [review]

**Pull request merged**